mix.exs

```diff
@@ -42,17 +42,17 @@ defmodule Guardian.Mixfile do
   end
 
   defp deps do
-    [{:jose, "~> 1.8"},
-     {:phoenix, "~> 1.2.0", optional: true},
-     {:plug, "~> 1.0"},
+    [{:jose, "~> 1.8.1"},
```

Comment:
@alexandrubagu do we need to include the patch version?  `1.8` should cover `1.8.x`